storage/client/blob/CHANGELOG.md

```diff
@@ -7,10 +7,10 @@ https://aka.ms/azure-sdk-preview1-java.
 
 **Breaking changes: New API design**
 - Operations are now scoped to a particular client:
-    - `StorageClient`: This client handles account-level operations. This includes managing service properties and listing the containers within an account.
-    - `ContainerClient`: The client handles operations for a particular container. This includes creating or deleting that container, as well as listing the blobs within that container and managing properties and metadata.
-    - `BlobClient`: The client handles operations for a particular blob. This includes creating or deleting that blob, as well as upload and download data and managing properties.
-    This BlobClient handles all blob types (block, page and append). Where operations can behave differently according to type (i.e. `upload`) the default behaviour will be block blobs unless otherwise specified.
+    - `StorageClient`: The StorageURL renamed to StorageClient. This client handles account-level operations. This includes managing service properties and listing the containers within an account.
```

Comment:
For the first sentence I would state "StorageURL's functionality was migrated to StorageClient" since StorageURL wasn't purely renamed to StorageClient.

Same comment goes for ContainerURL -> ContainerClient and BlobURL -> BlobClient.

Reply:
Done.